terra-jupyter-python/tests/smoke_test.ipynb

```diff
@@ -85,9 +85,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "output = !pip3 show pendulum\n",
+    "output = !pip3 show emoji\n",
     "print(output)  # Should show not yet installed.\n",
-    "assert(0 == output.count('Name: pendulum'))"
+    "assert(0 == len([i for i in output if 'Name: emoji' in i]))"
```

Comment:
Needed to adjust this because the output strings now contain some color-coding junk characters. We need to check whether the strings *contain* the expected message rather than being equal to it.